src/ini/config.c

```diff
@@ -0,0 +1,79 @@
+#include "config.h"
+#include "ini.h"
+#include <string.h>
+
+int match(const char *section, const char *check_section, const char *name, const char *check_name) {
+        if (strcmp(section, check_section) == 0 && strcmp(name, check_name) == 0) {
+                return 1;
+        }
+        return 0;
+}
+
+/*
+This function will be used in ini_parse() for node ini file parsing.
+If 0 return that means that the ini file is not in the right schem
+*/
+static int handler_node(void *user, const char *section, const char *name, const char *value) {
+        configurationNode *pconfig = (configurationNode *) user;
+
+        if (match(section, "Node", name, "OchestratorPort")) {
+                pconfig->ochestrator_port = strdup(value);
```

Comment:
Rather than ```strdup()```, use ```strndupa_safe()```.  We can use something like systemd has, in src/basic/alloc-util.h.